magento/magento2
# Allow sending email to BCC address when not notifying user.

### Description (*)

Order Comments Email Copy would NOT be sent to the specified email address if the send method is set to `Bcc`, and `Notify Customer by Email` is not checked when adding a comments from backbend.

The expected behaviour is that whether the `Notify Customer by Email` is checked, an email copy is sent to the specified email. While now if the `Send Order Comments Email Copy Method` config is set to `Separate Email` it goes well. But if the config is set to `Bcc`, the email can not be sent.

The problem is that the `SenderBuilder` is used by both order notification and comments notification, so the fix is to add a flag to fix the problem for comments notification.

### Fixed Issues (if relevant)



### Manual testing scenarios (*)

1. Navigate to Configuration->SALES->Sales Emails->Order Comments
2. Input email address in `Send Order Comment Email Copy To`
3. Select Bcc for `Send Order Comments Email Copy Method`.
4. Go to any order, add a new comment to the order and leave the `Notify Customer by Email` unchecked.
3. Verify the email address set in Step 2 can receive email after submitting the comment.

### Contribution checklist (*)
